pyproject.toml

```diff
@@ -79,7 +79,7 @@ exclude = [
     "otterdog/app.py",
     "otterdog/webapp"
 ]
-version = "0.0.0"
+version = "1.0.0"
```

Comment:
the version is computed [dynamically](https://github.com/eclipse-csi/otterdog/pull/393/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R34), so I think it's best practice to leave this to `0.0.0` to ensure that the version is set via poetry.

@netomi, do you confirm?

Reply:
I have 
```
❯ otterdog --version
otterdog.sh, version 1.0.0
```

:D

Reply:
my guess is that your `otterdog` binary is from local build of the repo, while mine is from pipx install. what is the output of `which otterdog`?

Reply:
so this is returned for me:

```
tn@proteus:~/workspace/eclipse/otterdog$ otterdog --version
otterdog.sh, version 1.1.0.dev3
```

Reply:
so the idea is that 1.0.0 was the last released version.
We are now 3 commits ahead of that and this bump the version for the next release that we aim for which is to bump the minor number.

All of this can be adapted as needed, but it made the most sense to me.

Reply:
Thanks!

@mrybczyn, did you figure out how you got `0.0.0`?

Reply:
@mbarbero source build (not pip) at the main branch (not tag)

On the freshly cloned repo it i still doing it -> and makes sense if dynamic versioning works on tag/packaging only

Reply:
Thanks! Then maybe revert this second commit and keep only the edit to the README?

Reply:
Yep will keep update it. We might mention the versioning in doc, but will make a SEPARATE PR for it :)

Reply:
Updated.